devito/symbolics/printer.py

```diff
@@ -115,6 +117,30 @@ def _print_Abs(self, expr):
         func = "abs" if is_integer else "fabs"
         return "%s(%s)" % (func, self._print(expr.args[0]))
 
+    def _print_Add(self, expr, order=None):
+        """"
+        Lifted from sympy's PR #24030 that fixes the _print_Add in sympy 1.11
+
+        Because we want to be compatible with sympy 1.11 we need to
+        keep the implementation here.
+        """
+        terms = self._as_ordered_terms(expr, order=order)
+
+        PREC = precedence(expr)
+        l = []
+        for term in terms:
+            t = self._print(term)
+            if precedence(term) < PREC:
+                l.extend(["+", "(%s)" % t])
```

Comment:
@FabioLuporini that's the part I don't like in sympy. They have
```
sign = "-" if t.startswith("-") else "="
l.extend([sign, "(%s)" % t[1:]])
```
Which is weird. Because they assume you can refactorize `-xxxxxx` into `-(xxxxxx)` no matter what `t` is the precedence is lower. I can't find a quick erroring case but it looks wrong.

Reply:
I see. Can you open an issue and/or propose a PR? 

We can give them a day or two to respond, if they don't, we merge ours? 

And if this procrastinate the sympy syncing (because e.g. a further patch would have to wait for 1.13) then we can merge ours regardless

Reply:
Yeah gonna try to make an MFE, I think they merged a PR that fixes the MFE but in a weird way will let you know